docs/v0.7/en/greptimecloud/usage-&-billing/dedicated.md

```diff
@@ -0,0 +1,34 @@
+# Dedicated Plan
+
+The dedicated plan offers unlimited data storage and retention,
```

Comment:
Co @sunng87's comment, the dedicated plan can start with:

The dedicated plan allows you to buy dedicated CPUs and storage to host GreptimeDB. It's up to unlimited data storage and retention, ...